homework.md

```diff
@@ -4,110 +4,111 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240
 
-## Question 4 
+## Question 4
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+CSS customizes the appearance and the layout of the webpage.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
-
+h1 {
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language to make webpages interactive.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1. keep track of the history of the changes. it is simply used for version control.
+2. allow multiple people do their own changes and merge them into one source.
+3. allow you revert to specific versions should you need to.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You copy the project and make changes without affecting the original project.
 
-## Question 10 
+## Question 10
 
-What happens when you clone a repostory?
+What happens when you clone a repository?
 
-It send it to a friend
+You copy an existing repository to your local computer.
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+it notifies the author of the repository that you have completed some tasks.
```

Comment:
We are using Pull Requests in this module to notify that we have completed some tasks. That is correct :+1: 

On a more general use case, when would we use a Pull Request? :)